solutionbox/structured_data/mltoolbox/_structured_data/master_setup.py

```diff
@@ -60,7 +76,7 @@ def get_version():
     'tensorflow==1.0',
     'protobuf==3.1.0',
     'google-cloud-dataflow==0.5.5',
-    #'psutil>=4.3.0'
+    'psutil'
```

Comment:
ah, I did, and when I built it copied the file over. But I forgot to re-build it.